Setup snmp-notifier 7.1 container in konflux

sachin-yeshwanth11 opened the pull request:
Hello @andrewschoen ,

This PR contains changes required for setting up snmp-notifier 7.1 container in Konflux.

Please help in reviewing and merging this PR, Thanks !

Commit:
setup snmp-notifier 7.1 container in konflux

sachin-yeshwanth11 commented:
/retest

andrewschoen commented on .tekton/snmp-notifier-7-1-pull-request.yaml:
```diff
@@ -24,12 +24,15 @@ spec:
   - name: build-platforms
     value:
     - linux/x86_64
+    - linux/arm64
```
This can be removed, we don't build for `arm64` until 8.1

sachin-yeshwanth11 replied:
I have removed `arm64` here

andrewschoen commented on .tekton/snmp-notifier-7-1-push.yaml:
```diff
@@ -23,10 +23,13 @@ spec:
   - name: build-platforms
     value:
     - linux/x86_64
+    - linux/arm64
```
can also be removed here

sachin-yeshwanth11 replied:
removed here as well

andrewschoen reviewed (changes requested):
This looks great. I think removing `arm64` is all we need to do here. Thanks!

andrewschoen commented on .tekton/snmp-notifier-7-1-pull-request.yaml:
```diff
@@ -24,12 +24,15 @@ spec:
     value: '{{revision}}'
   - name: output-image
-    value: quay.io/rhceph-dev/snmp-notifier:on-pr-{{revision}}
+    value: quay.io/rhceph-ci/snmp-notifier:on-pr-{{revision}}
```
please change the tag to be `on-pr-v7.1-{{revision}}`

sachin-yeshwanth11 replied:
updated the tag

andrewschoen commented on .tekton/snmp-notifier-7-1-push.yaml:
```diff
@@ -23,10 +23,13 @@ spec:
     value: '{{revision}}'
   - name: output-image
-    value: quay.io/rhceph-dev/snmp-notifier:{{revision}}
+    value: quay.io/rhceph-ci/snmp-notifier:{{revision}}
```
please change the tag to be `v7.1-{{revision}}`

sachin-yeshwanth11 replied:
updated here as well

andrewschoen commented:
> This looks great. I think removing `arm64` is all we need to do here. Thanks!

Could you also please add the additional tagging to this PR as well? https://github.com/ibmstorage/snmp-notifier-container/blob/release-5.3/.tekton/snmp-notifier-5-3-pull-request.yaml#L559

sachin-yeshwanth11 commented:
/retest

sachin-yeshwanth11 commented:
> > This looks great. I think removing `arm64` is all we need to do here. Thanks!
> 
> Could you also please add the additional tagging to this PR as well? https://github.com/ibmstorage/snmp-notifier-container/blob/release-5.3/.tekton/snmp-notifier-5-3-pull-request.yaml#L559

Hello Andrew - I have addressed all the review points made by you, please help to re-review. Thanks !
